### PR TITLE
Swig enhancements

### PR DESCRIPTION
**Pull request by dalexander**
This is some work to make the SWIG interface a little more usable

### CODE REVIEW

**Comment by dalexander** [comment]
...rebased

File: swig/ConsensusCore2.i [hunk 1]
```diff
@@ -7,11 +7,6 @@
 #define SWIG_FILE_WITH_INIT
 %}
 
-%pythoncode
-%{
-__version__ = "@UNANIMITY_VERSION@"
-%}
-
```
Comment (nlhepler) [inline swig/ConsensusCore2.i]: why are you getting rid of this?

File: swig/Version.i.in [hunk 2]
```diff
@@ -3,4 +3,9 @@
 #include <pacbio/Version.h>
 %}
 
+%pythoncode
+%{
+__version__ = "@UNANIMITY_VERSION@"
+%}
+
```
Comment (nlhepler) [inline swig/Version.i.in]: oh I see, nevermind.

File: src/matrix/ScaledMatrix.cpp [hunk 3]
```diff
@@ -80,5 +80,19 @@ std::ostream& operator<<(std::ostream& os, const ScaledMatrix& mat)
...
+void ScaledMatrix::ToHostMatrix(double** mat, int* rows, int* cols) const
+{
+    const double nan = std::numeric_limits<double>::quiet_NaN();
+    *mat = new double[Rows() * Columns()];
+    *rows = Rows();
+    *cols = Columns();
+    for (size_t i = 0; i < Rows(); i++) {
+        for (size_t j = 0; j < Columns(); j++) {
+            (*mat)[i * Columns() + j] =
+                    IsAllocated(i, j) ? std::log(Get(i, j)) + GetLogScale(j) : nan;
```
Comment (nlhepler) [inline src/matrix/ScaledMatrix.cpp]: not seeing counterWeight anywhere..?

File: swig/Version.i.in (hunk 2 above)
Comment (nlhepler) [inline swig/Version.i.in]: Actually, this can be __version__ = UnanimityVersion() without every having to configure.

**Comment by dalexander** [comment]
I have not yet made any changes there.  That's a separate PR I'm working on.

File: swig/Version.i.in (hunk 2 above)
Comment (nlhepler) [inline swig/Version.i.in]: let me do this in my fixPython branch, can you remove the move from ConsensusCore.i.in to ConsensusCore.i and this version shift? I'll take care of it there.

**Comment by dalexander** [comment]
Lance, I'm going to check this in as-is but you can trivially change the Version.i file to be unconfigured and use the UnanimityVersion()

**Pull request merged**